frontend/src/app/index.tsx

```diff
@@ -72,6 +72,13 @@ const App = () => {
     if (!data) return;
 
     const getDefaultFacility = () => {
+      const urlFacilityId = window.location.pathname.split("/")[2];
```

Comment:
this is brittle af; getting the facilityId from localstorage would be better than from urls

Reply:
Yeah this does seem pretty flakey and parsing URLs yourself is always a dicey thing to do from a security standpoint as well. So if we know the facility for the currently logged in user, can we just keep it in memory? If they refresh the page we'll still need to do the `whoami` to know they're logged in, and we can re-set the facility then.

Reply:
We don't know the facility of the user right now as far as I am aware of, and a user might be associated with multiple facilities I'd imagine. I suppose the other options here are to:

1. store the current facility the user is focused on in the db. In the whoami query, get the current facility. 
2. store the current facility id in localstorage. When refreshing the page, load it from localstorage. If the user changes the facility id, update localstorage. I think this is the better solution
3. store the facility id in the url (this PR). If the user refreshes the page, it will grab the id in the URL. If the user changes the target facility, the url changes.

Reply:
I think option 3 is the only option as it will allow for links to behave as expeced.

Example:
User bookmarks a page while viewing Facility A. That bookmark should point to Facility A even if the user changes there facility to Facility B

Reply:
For this manipulation, how about if we have a utility function do this rather than the split/reassemble? That way all the URL munging is in one place.